### DISCUSSION TITLE
Dynamic number of side by side installation

### DISCUSSION THREAD

**Discussion by AnthonyFontaine-ES** [wixtoolset/issues#8798] [Questions]
Hey, 

I'm working on a project that involves creating a Windows service application. This service will run on a server and be responsible for reading data from a source and pushing it to a local queuing system.

To ensure a smooth and efficient installation process for users, I'm planning to use Wix to create an installer for the service.

A key feature of this service is its ability to have multiple instances running concurrently on the same server, each with its own independent configuration set. The Wix installer would need to accommodate this by allowing users to easily configure and manage these instances during the installation process.

I'd appreciate any insights or guidance you can offer regarding the use of Wix for creating installers for this type of application. Specifically, I'm interested in best practices for handling the configuration of multiple service instances during installation and if this could be done using Wix.

Thanks in advance for your time and assistance.

**Comment by robmen** [comment]
That's an advanced Windows Installer topic, start here: [Installing Multiple Instances of Products](https://learn.microsoft.com/en-us/windows/win32/msi/installing-multiple-instances-of-products-and-patches)
